src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -59,16 +58,16 @@
 import chip.clusters as Clusters
 import chip.logging
 import chip.native
-from chip import discovery
+import chip.testing.global_stash as global_stash
 from chip.ChipStack import ChipStack
 from chip.clusters import Attribute
 from chip.clusters import ClusterObjects as ClusterObjects
```

Comment:
Same question here.
Would this work? Or do you need `as ClusterObjects`?

```suggestion
from chip.clusters import ClusterObjects
```

Reply:
Removed `as ClusterObjects` and rebuilt. No issues found on commissioning.